WORKSPACE

```diff
@@ -311,55 +311,55 @@ http_archive(
 http_archive(
     name = "nodejs10_amd64",
     build_file = "//nodejs:BUILD.nodejs",
-    sha256 = "aa7e9e1d8abcc169119bf5c56ede515689f2644ccc4d40ca0fc33756a3deb1f7",
+    sha256 = "19cccb78f0881a78051291a50200200a0303649ee84e5489c771d3b4e4bd0e51",
     strip_prefix = "node-v10.22.0-linux-x64/",
```

Comment:
You should fix this line too. (It is the top-level directory name to strip for the files in the tarball.)

Reply:
fixed, thanks for pointing it out!